### PR TITLE
Tdr 334 add file format resources

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: modules/shared-vpc/root.tf [hunk 1]
```diff
@@ -3,7 +3,8 @@ data "aws_availability_zones" "available" {
 
 resource "aws_vpc" "main" {
   cidr_block = "10.0.0.0/16"
-
+  enable_dns_hostnames = true
```
Comment (TomJKing) [inline modules/shared-vpc/root.tf:6]: Has terraform fmt been run?

File: root.tf [hunk 2]
```diff
@@ -309,3 +311,33 @@ module "api_update_lambda" {
...
+  file_system                           = module.file_format_efs.file_system
+  file_format_efs_access_point          = module.file_format_efs.access_point
+  vpc_id                                = module.shared_vpc.vpc_id
+}
+
+module "file_format_efs" {
+  source            = "./tdr-terraform-modules/efs"
+  common_tags       = local.common_tags
+  function          = "file-format-efs"
+  project           = "tdr"
```
Comment (TomJKing) [inline root.tf:332]: Should this be var.project instead of the hardcoded tdr value?

Comment (MancunianSam) [reply root.tf:332]: Yep, you're right, I've changed it.

File: modules/shared-vpc/root.tf (hunk 1 above)
Comment (MancunianSam) [reply modules/shared-vpc/root.tf:6]: I thought I had, I missed this one.

**Pull request merged**